src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -103,409 +106,276 @@
-              .withPosition(7, 0)
-              .getEntry();
-      sbHaOneI =
-          climbTab
-              .add("Ha1 I", Constants.HaRightPID[1])
-              .withSize(1, 1)
-              .withPosition(8, 0)
-              .getEntry();
-      sbHaOneD =
-          climbTab
-              .add("Ha1 D", Constants.HaRightPID[2])
-              .withSize(1, 1)
-              .withPosition(9, 0)
-              .getEntry();
-
-      // High Arm 2
-      sbHaHeightTwo = climbTab.add("Ha2 Height", 0).withSize(2, 1).withPosition(0, 1).getEntry();
-      sbHaTargettedTwo =
-          climbTab.add("Ha2 targetted", 0).withSize(3, 1).withPosition(2, 1).getEntry();
-      sbHaSpeedTwo = climbTab.add("Ha2 Speed", 0).withSize(2, 1).withPosition(5, 1).getEntry();
-      sbHaTwoP =
-          climbTab
-              .add("Ha2 P", Constants.HaLeftPID[0])
-              .withSize(1, 1)
-              .withPosition(7, 1)
-              .getEntry();
-      sbHaTwoI =
-          climbTab
-              .add("Ha2 I", Constants.HaLeftPID[1])
-              .withSize(1, 1)
-              .withPosition(8, 1)
-              .getEntry();
-      sbHaTwoD =
-          climbTab
-              .add("Ha2 D", Constants.HaLeftPID[2])
-              .withSize(1, 1)
-              .withPosition(9, 1)
-              .getEntry();
-
-      // Other
-    } else {
-      sbMcHeightOne =
-          operatorTab
-              .add("Mc Height 1", 0)
-              .withWidget(BuiltInWidgets.kNumberBar)
-              .withSize(2, 1)
-              .withPosition(6, 0)
-              .getEntry();
-      sbMcHeightTwo =
-          operatorTab
-              .add("Mc Height 2", 0)
-              .withWidget(BuiltInWidgets.kNumberBar)
-              .withSize(2, 1)
-              .withPosition(6, 1)
-              .getEntry();
-      sbHaHeightOne =
-          operatorTab
-              .add("Ha Height 1", 0)
-              .withWidget(BuiltInWidgets.kNumberBar)
-              .withSize(2, 1)
-              .withPosition(6, 1)
-              .getEntry();
-      sbHaHeightTwo =
-          operatorTab
-              .add("Ha Height 2", 0)
-              .withWidget(BuiltInWidgets.kNumberBar)
-              .withSize(2, 1)
-              .withPosition(6, 1)
-              .getEntry();
-    }
+  public void deployHooks() {
+    // servoOne.set(Constants.climbServoSetPoint);
+    // servoTwo.set(Constants.climbServoSetPoint);
   }
 
-  @Deprecated
-  public void runManual() {
-    if (climbEnabble) {
+  public boolean atFirstSetpoint() {
+    // checking if the armHeight is within an acceptable range
+    return (armEncoderHeightOne > Constants.armHeightMin - Constants.climbArmDeadband
+            && armEncoderHeightOne < Constants.armHeightMin + Constants.climbArmDeadband)
+        && (armEncoderHeightTwo > Constants.armHeightMin - Constants.climbArmDeadband
+            && armEncoderHeightTwo < Constants.climbArmDeadband + Constants.climbArmDeadband);
+  }
```

Comment:
This is the end condition of the automatic deploy.

It feels like you could get caught in an edge case with each arm trying to get within the correct range.
It might not be a big problem with the only opposing forces being the gas springs.  

Should check during testing and see if the position for either arm oscillates around your armHeightMin value, and if so, do something here to let it track each arm independently.

Reply:
Got it, I'll keep a look as to how this acts during testing.